### PR TITLE
Add new documentation tool: openAPI-UI

### PR DESCRIPTION
**Pull request by rookie-luochao**
Thanks for this project

This PR adds a new documentation tool that  show api, it is simpler and more beautiful than swagger UI, it can quickly generate mock params and call api request

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -256,6 +256,18 @@
   v3: true
   v3_1: true
 
+- name: OpenAPI-UI
+  category: documentation
+  description:
+    a simpler and more beautiful openapi UI document than swagger UI, 
+    quickly generate mock params and call api request.
```
Comment (AndrewWalsh) [inline _data/tools.yml:263]:
```suggestion
    Create simple and beautiful OpenAPI/Swagger documentation from OpenAPI files.
    Generate mock parameters and call APIs. Like Postman, but for OpenAPI specifications.
```

File: _data/tools.yml [hunk 2]
```diff
@@ -256,6 +256,18 @@
...
+  link: https://docs.openapi-ui.com
+  github: https://github.com/rookie-luochao/openapi-ui
+  language: React.js
+  v2: true
+  v3: true
+  v3_1: true
```
Comment (AndrewWalsh) [inline _data/tools.yml:269]: Can you confirm it 100% works with the `3.1` standard?

File: _data/tools.yml [hunk 3]
```diff
@@ -256,6 +256,18 @@
...
   v3_1: true
 
+- name: OpenAPI-UI
+  category: documentation
+  description:
+    a simpler and more beautiful openapi UI document than swagger UI, 
+    quickly generate mock params and call api request.
+  link: https://docs.openapi-ui.com
+  github: https://github.com/rookie-luochao/openapi-ui
+  language: React.js
```
Comment (AndrewWalsh) [inline _data/tools.yml:266]:
```suggestion
  language: TypeScript
```

File: _data/tools.yml [hunk 4]
```diff
@@ -256,6 +256,18 @@
   v3: true
   v3_1: true
 
+- name: OpenAPI-UI
+  category: documentation
+  description:
+    a simpler and more beautiful openapi UI document than swagger UI, 
+    quickly generate mock params and call api request.
+  link: https://docs.openapi-ui.com
```
Comment (AndrewWalsh) [inline _data/tools.yml:264]: Great feature, just a suggestion: a default OpenAPI spec in 2.0, 3.0, 3.1 standards would make it easier to see what your tool does.

**Review by AndrewWalsh — COMMENTED** [review]
This looks really impressive, great work. Please inspect this review @rookie-luochao . 

Two thoughts:
- Would be great to see this in use instantly, with default OpenAPI specs on your main site (without users having to upload their own). That way, users can see what your UI/tool does and looks like without effort.
- An English translation would be appreciated.

**Comment by rookie-luochao** [comment]
> This looks really impressive, great work. Please inspect this review @rookie-luochao .
> 
> Two thoughts:
> 
> * Would be great to see this in use instantly, with default OpenAPI specs on your main site (without users having to upload their own). That way, users can see what your UI/tool does and looks like without effort.
> * An English translation would be appreciated.

* It provides multiple themes and nationalization functions, supporting English
* It supports backend framework access. I have already supported some [golang frameworks](https://github.com/openapi-ui/go-openapi-ui) and [nodejs frameworks](https://github.com/openapi-ui/nodejs-openapi-ui).

I don't know if your suggestion is the above two, Welcome to communicate

File: _data/tools.yml (hunk 2 above)
Comment (rookie-luochao) [reply _data/tools.yml:269]: support for 3.1 is not comprehensive and needs to be improved while using it

Comment (AndrewWalsh) [reply _data/tools.yml:269]: Ok I advise setting this to be false for now, can always submit a new PR to update this later.

File: _data/tools.yml (hunk 4 above)
Comment (AndrewWalsh) [reply _data/tools.yml:264]: I was just offering a suggestion, purely my own, that having default specs in various OpenAPI versions that users could just click on would let your users see instantly how your tool works. Just food for thought, and nothing more.

File: _data/tools.yml (hunk 2 above)
Comment (rookie-luochao) [reply _data/tools.yml:269]: This is a good suggestion, Could you please provide a Commit suggestion? Thanks

File: _data/tools.yml [hunk 5]
```diff
@@ -256,6 +256,18 @@
...
+  category: documentation
+  description:
+    Create simple and beautiful OpenAPI/Swagger documentation from OpenAPI files.
+    Generate mock parameters and call APIs. Like Postman, but for OpenAPI specifications.
+  link: https://docs.openapi-ui.com
+  github: https://github.com/rookie-luochao/openapi-ui
+  language: TypeScript
+  v2: true
+  v3: true
+  v3_1: true
```
Comment (AndrewWalsh) [inline _data/tools.yml:269]: Sure thing.

```suggestion
  v3_1: false
```

File: _data/tools.yml (hunk 2 above)
Comment (AndrewWalsh) [reply _data/tools.yml:269]:
```suggestion
  v3_1: false
```

File: _data/tools.yml (hunk 4 above)
Comment (rookie-luochao) [reply _data/tools.yml:264]: Maybe I should automatically fill in an openapi-spec to support users to quickly pre-render?

**Comment by AndrewWalsh** [comment]
> > This looks really impressive, great work. Please inspect this review @rookie-luochao .
> > Two thoughts:
> > 
> > * Would be great to see this in use instantly, with default OpenAPI specs on your main site (without users having to upload their own). That way, users can see what your UI/tool does and looks like without effort.
> > * An English translation would be appreciated.
> 
>     * It provides multiple themes and i18n functions, supporting English
> 
>     * It supports backend framework access. I have already supported some [golang frameworks](https://github.com/openapi-ui/go-openapi-ui) and [nodejs frameworks](https://github.com/openapi-ui/nodejs-openapi-ui).
> 
>     * I provide a web page to quickly pre-render and use OpenAPI-UI, but it also supports web framework access through middleware
> 
> 
> I don't know if your suggestion is the above two, Welcome to communicate

Nothing I said in this regard is a suggestion, and I haven't delved into your tool enough to assess. It bears no relation to the review, so no worries!

File: _data/tools.yml (hunk 4 above)
Comment (AndrewWalsh) [reply _data/tools.yml:264]: I was thinking just having PetStore-default OpenAPI specs in various versions available on your site. Then users could click on it (without the bother of having their own, uploading etc) and see what your tool looks like instantly.

This is completely off-topic and just a personal suggestion. It has no relation to this review, please disregard or factor as you wish :).

**Comment by rookie-luochao** [comment]
Thank you for your suggestion

File: _data/tools.yml (hunk 4 above)
Comment (rookie-luochao) [reply _data/tools.yml:264]: I understand what you mean. There is actually a link to the openapi-spec in the form-label. Maybe I should fill it in by default

**Comment by rookie-luochao** [comment]
Can this PR be merged?

**Comment by AndrewWalsh** [comment]
@rookie-luochao yes - thanks for your patience and congrats on your project 🎉

**Comment by rookie-luochao** [comment]
I should be the one thanking you. Thank you.

**Pull request merged**